rebabel_format/converters/nlp_part_of_speech.py

```diff
@@ -18,6 +18,13 @@ class NLPPartOfSpeechReader(LineReader):
 
     identifier = "nlp_pos"
 
+    def __init__(self, db, user, conf, kwargs):
```

Comment:
```python
identifier = "nlp_pos"
delimiter = Parameter(...)

def reset....
```

Reply:
as it currently stands, this invocation of `Parameter` has no effect. `Parameter` is only useful if it is outside of a function at the class level so the metaclass can process it properly

Reply:
and if it's at the class level, then an explicit `__init__` should be unnecessary.